internal/projects/bgpsessions.go

```diff
@@ -41,19 +41,17 @@ func (c *Client) BGPSessions() *cobra.Command {
   metal project bgp-sessions --project-id 50693ba9-e4e4-4d8a-9eb2-4840b11e9375`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			listOpt := c.Servicer.ListOptions(nil, nil)
-			getOpts := &packngo.GetOptions{Includes: listOpt.Includes, Excludes: listOpt.Excludes}
-			p, _, err := c.BGPConfigService.Get(projectID, getOpts)
+			p, _, err := c.BGPConfigService.FindProjectBgpSessions(context.Background(), projectID).Execute()
```

Comment:
Judging from `packngo`, BGP sessions are paginated: https://github.com/packethost/packngo/blob/master/projects.go#L154-L175

The spec doesn't say it's paginated, so we'll have to address that in order to handle pagination for BGP settings: https://deploy.equinix.com/developers/api/metal#tag/BGP/operation/findProjectBgpSessions

Reply:
Then, Is it fine to close conversation?

Reply:
Yeah, this is not an issue after all